src/evaluate/evaluator.py

```diff
@@ -88,7 +94,8 @@ def _compute_predictions(self, pipe: Pipeline, inputs, label_mapping: Dict[str,
 
     def compute(
         self,
-        model_or_pipeline=None,
+        model_or_pipeline: Union[str, Pipeline, Callable, PreTrainedModel, TFPreTrainedModel] = None,
+        tokenizer: Optional[Union[str, PreTrainedTokenizer]] = None,
```

Comment:
Just a nit: can we move the tokenizer after data/metric. I think that conveys a bit better that in most cases you just need to provide model/data/metric and not a tokenizer.

Reply:
sounds good, basically on a small example I tested the tokenizer wasn't compatible, that's why I put it here, but happy to remove for clarity.

Reply:
I mean we can keep it but just change the orderings of the arguments :)